src/themes/types.js

```diff
@@ -678,6 +678,7 @@ export type GridT = {
   margins: ResponsiveT<number>,
   gaps: ResponsiveT<number>,
   maxWidth: number,
+  unit: string,
```

Comment:
I wonder if we should limit this type to a list of supported CSS units.

Reply:
yes, I believe the same. what are the CSS units should we support?

Reply:
We can just add everything [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#Lengths). The `Block` component does (something similar)[https://github.com/uber/baseweb/blob/master/src/block/types.js#L19-L232].

Some of these units don't totally make sense but doesn't hurt to include them for completeness.

Reply:
done 👍